proptest-attributes/tests/compile-fail/method.stderr

```diff
@@ -0,0 +1,11 @@
+error: The #[proptest] macro cannot be applied to a method.
+  --> $DIR/method.rs:17:21
+   |
+17 |     fn function_arn(&self) {
+   |                     ^^^^^
+
+error[E0601]: `main` function not found in crate `$CRATE`
```

Comment:
off-topic error, add `fn main() {}` to get rid of it. (we do this in the compiler)